eq-author/src/components/ContentPickerv2/index.js

```diff
@@ -0,0 +1,172 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+
+import styled from "styled-components";
+
+import Modal, { CloseButton } from "components/modals/Modal";
+import Button from "components/buttons/Button";
+import ButtonGroup from "components/buttons/ButtonGroup";
+
+import { colors } from "constants/theme";
+
+import { ANSWER, METADATA } from "components/ContentPickerSelect/content-types";
+import AnswerPicker from "./AnswerPicker";
+import MetadataPicker from "./MetadataPicker";
+
+const ModalFooter = styled.div`
+  padding: 1.5em;
+  border-top: 1px solid ${colors.bordersLight};
+`;
+
+export const StyledCloseButton = styled(CloseButton)`
+  position: absolute;
+  right: 0.5em;
+  top: 2em;
+  bottom: 0;
+  margin: auto;
+  align-items: center;
+`;
+
+const StyledModal = styled(Modal)`
+  .Modal {
```

Comment:
Is `.Modal` referencing itself? If so, could you do `& {...` or maybe just remove that selector?